src/config.js

```diff
@@ -2,24 +2,29 @@ const fs = require("fs").promises;
 
 const yaml = require("js-yaml");
 
-const { CONFIG_PATH } = require("./constants");
+const { CONFIG_PATH,
+  JPEG_QUALITY,
+  PNG_QUALITY,
+  WEBP_QUALITY,
+  IGNORE_PATHS
+} = require("./constants");
 
 const getYamlConfig = async () => {
   try {
     const text = await fs.readFile(CONFIG_PATH);
     return yaml.safeLoad(text);
   } catch (err) {
-    console.error(CONFIG_PATH, "not found", err);
+    console.error("::warning::", CONFIG_PATH, "not found", err);
```

Comment:
Seeing as the preexisting configuration method is only now implemented for backwards compatibility, let's remove this warning console error.